packages/arb-ts/src/lib/networks.ts

```diff
@@ -62,6 +63,7 @@ const networks: Networks = {
     partnerChainID: '42161',
     tokenBridge: mainnetBridge,
     blockTime: 15,
+    rpcURL: 'https://mainnet.infura.io/v3/' + process.env['INFURA_KEY'],
```

Comment:
I think we should pass in the whole rpc url as a env variable instead of just an infura key
Makes it more flexible

Reply:
same for rinkeby